src/ClientData/CaptureDataTest.cpp

```diff
@@ -55,10 +58,15 @@ constexpr uint64_t kSecondTid = 2000;
 constexpr uint64_t kNonExistingTid = 404;
 constexpr uint64_t kStTimestamp1 = 50;
 constexpr uint64_t kEnTimestamp1 = 100;
```

Comment:
Not part of this change, but do "st" and "en" mean "start" and "end"? Avoid custom abbreviations as per the style guide.